test/integration/instance/values.test.js

```diff
@@ -122,7 +122,7 @@ describe(Support.getTestDialectTeaser('DAO'), () => {
             // so we must create a record with the right value for always_false, then reference it in an update
             let now = dialect === 'sqlite' ? self.sequelize.fn('', self.sequelize.fn('datetime', 'now')) : self.sequelize.fn('NOW');
             if (dialect === 'mssql') {
-              now = self.sequelize.fn('', self.sequelize.fn('getdate'));
```

Comment:
Is this related to DATEONLY?

Reply:
Not really. I just noticed it when running tests locally and it was among the ones that failed along with DATEONLY tests due to timezone.